### ISSUE TITLE
Non-existing class used in RecaptchaEnterprise Facade on tagged branches

### ISSUE THREAD

**Issue by kovalovme** [oneduo/laravel-recaptcha-enterprise#27]
`\Oneduo\RecaptchaEnterprise\Facades\RecaptchaEnterprise` class contains `\Oneduo\RecaptchaEnterprise\Tests\Mocks\FakeRecaptchaEnterprise` import which does not exist on tagged branches, only in master

See line 11:

```
<?php

declare(strict_types=1);

namespace Oneduo\RecaptchaEnterprise\Facades;

use Closure;
use Illuminate\Support\Facades\Facade;
use Oneduo\RecaptchaEnterprise\Contracts\RecaptchaContract;
use Oneduo\RecaptchaEnterprise\Services\RecaptchaService;
> use Oneduo\RecaptchaEnterprise\Tests\Mocks\FakeRecaptchaEnterprise;

/**
 * @method static static setThreshold(float $threshold)
 * @method static static setScore(float $score)
 * @method static static setProperties(\Google\Cloud\RecaptchaEnterprise\V1\TokenProperties $properties)
 * @method static static forceValid(bool $value = true)
 *
 * @see RecaptchaService
 */
class RecaptchaEnterprise extends Facade
. . .
```

I'm using `barryvdh/laravel-ide-helper` and it throws an error on:

```
> Illuminate\Foundation\ComposerScripts::postUpdate
> @php artisan ide-helper:generate
```

Can you please remove usage or add mocks on tagged branches?
Thank you and have a nice day!

**Comment by Rezrazi** [comment]
Fixed in v0.4.2
Apologies for the oversight

**Comment by kovalovme** [comment]
Thank you very much!
